roles/reviewer/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: Install reviewer tool from git
+  git:
+    repo: "https://github.com/ares/reviewer.git"
```

Comment:
I'm a bit hesitant to add git repos from single developers and wonder if we should have it in ```theforeman``` namespace

Reply:
no problem with that from my side, though that could indicate that theforeman supports this script :-) again, no problem with that from my side :-) but perhaps we should wait until we get more feedback whether people find it useful

Reply:
Sure, let's start with it in the current form. If you could make this a parameter too then we've separated the logic from the data.